library/src/animationManager.cxx

```diff
@@ -55,7 +55,8 @@ bool animationManager::Initialize(
     progressRep->SetPadding(0.0, 0.0);
 #endif
 
-    this->ProgressWidget->On();
+    if (this->Options->getAsBool("ui.animation-progress"))
+      this->ProgressWidget->On();
```

Comment:
if (this->Options->getAsBool("ui.animation-progress"))
    {
      this->ProgressWidget->On();
    }

Reply:
actually this is incorrect. you should use setVisibility after turning the widget On iirc.

Reply:
> If I remember well, "Initialize" is called on loadFile, which makes this option updated on "load", which is fine :)

So it is okay to check this option only inside Initialize, but it should use SetVisibility instead of not turning the widget on, right?

Reply:
yes indeed.